src/solvers/flattening/boolbv_complex.cpp

```diff
@@ -54,7 +47,7 @@ bvt boolbvt::convert_complex_real(const complex_real_exprt &expr)
 
   bvt bv = convert_bv(expr.op());
 
-  assert(bv.size()==width*2);
+  INVARIANT(bv.size() == width * 2, "");
```

Comment:
Missing message, though really you should just drop it and add the extra argument to `convert_bv`?